### DISCUSSION TITLE
Improve device accessibility

### DISCUSSION THREAD

**Discussion by protyposis** [home-assistant/frontend#18961] [Accessibility]
My use of the HA UI is quite device-centered, so I often find myself navigating to devices (the "device info" view). Typical use-cases are:
- identify the area an entity belongs to
- view a related entity (e.g. check the battery level of a temperature sensor)
- view related automations

However, the dashboard and UI in general seem entity-focused:
- The quick bar only searches entities
- The dashboards only present entities

So to get to a device, I need to navigate
- Settings > Devices > Devices [> Search] > Select device
- or open an entity view > Menu > Device info

Interestingly, the entity settings, which I find myself use much more seldomly, are directly linked in the entity view (the cog button). I understand the implementation perspective that settings are part of the same view/dialog, while the device view isn't, but from a usability perspective this seems counterintuitive to me. Am I using the UI wrongly, am I missing something, or could this be improved? I don't see the "Related" view as a viable alternative since it's as complicated to reach, but displays less information (no states).

I'd like to suggest the following improvements:
- Display area name directly in the entity view dialog to make it immediately obvious where an entity belongs (if "Use device area" is selected). (I think this could also help with device naming, as I could e.g. name the "Office desk lamp" simply "Desk lamp", and I think this could even be generally displayed with entity names across the UI.)
- Move the "Device info" button out of the menu to make it more obvious that an entity belongs to a device, and save one navigational click.
- (Allow searching devices in the quick bar?)

Looking forward to your opinion!

**Comment by shbatm** [comment]
I would also love to be able to search and navigate directly to a device using the Quick Bar, maybe adding a `d` shortcut key to be able to search specifically for devices and navigate to that page?
